services/director-v2/src/simcore_service_director_v2/models/schemas/comp_scheduler.py

```diff
@@ -0,0 +1,21 @@
+from models_library.projects_nodes_io import NodeID
+from pydantic import BaseModel
+
+from ..domains.comp_tasks import Image
+
+
+class TaskIn(BaseModel):
+    node_id: NodeID
+    runtime_requirements: str
```

Comment:
MINOR: regex? ``(\bgpu\b|\bmpi\b):{0,1}``  see [regex test](https://pythex.org/?regex=(%5Cbgpu%5Cb%7C%5Cbmpi%5Cb)%3A%7B0%2C1%7D&test_string=test%3Afoo%3A%0Agpu%3Ampi%0Ampi&ignorecase=0&multiline=0&dotall=0&verbose=0)

Reply:
as discussed. for later